homeassistant/components/media_player/liveboxplaytv.py

```diff
@@ -50,7 +50,11 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     port = config.get(CONF_PORT)
     name = config.get(CONF_NAME)
 
-    add_devices([LiveboxPlayTvDevice(host, port, name)], True)
+    try:
+        add_devices([LiveboxPlayTvDevice(host, port, name)], True)
+    except IOError:
+        _LOGGER.error('Failed to connect to Livebox Play TV at %s:%s. '
+                      'Please check your configuration.', host, port)
```

Comment:
:+1:  you can also call update after init for check and after that call `add_devices` without boolean at the end.